bite-size-notes/week-8-b-probability-review.md

```diff
@@ -51,6 +54,50 @@ $$P(x,y,z) = P(x \mid y,z) P(y,z) = P(x \mid y,z) P(y \mid z) P(z)$$
 
 **Bayes Rule**: Bayes rule allows us to write the conditional probability of one variable given another based on the reverse conditional probability and probability functions of the individual RVs. The general form of Bayes rule is: 
 $$P(x \mid y) = \frac{P(y \mid x) P(x)}{P(y)}$$
-Where $P(y) = \int_x P(y \mid x) P(x) dx$ and can be considered a sort of scaling or normalizing function. In practice, $P(y)$ can often be intractable or too computationally expensive to compute so approximations methods can be used. Conceptually, we can describe the terms in Bayes rule as follows: $P(x \mid y) is the posterior, so if $y$ is a measurement of $x$, then what is $x$ given the measurement? $P(x)$ is the prior, which describes the belief or guess of $x$ before the measurement is known or aquired. $P(y \mid x)$ is the likelihood function or measurement function, this describes the expected measurement $y$ given the state $x$ (if you knew $x$, what measurement $y$ would you expect to see?)
+Where $P(y) = \int_x P(y \mid x) P(x) dx$ and can be considered a sort of scaling or normalizing function. In practice, $P(y)$ can often be intractable or too computationally expensive to compute so approximations methods can be used. Conceptually, we can describe the terms in Bayes rule as follows: $P(x \mid y)$ is the posterior, so if $y$ is a measurement of $x$, then what is $x$ given the measurement? $P(x)$ is the prior, which describes the belief or guess of $x$ before the measurement is known or aquired. $P(y \mid x)$ is the likelihood function or measurement function, this describes the expected measurement $y$ given the state $x$ (if you knew $x$, what measurement $y$ would you expect to see?)
 
 **State Estimation**: Bayes rule naturally leads us to the concept of state estimation, which aims to answer the question poised by Bayes rule: What is the probability of you being in state $x$ given your measured value of the state $y$? We know how to calculate this based on $P(y \mid x)$, $P(x)$, and $P(y)$, but what if we have multiple measurements? This gives rise to the recursive Bayes estimator, which aims to estimate the true state of a dynamical system based on a sequence of observations. At this point, you should be have all of the basics of probability readily available and can move on to applications such as the Recursive Bayes Estimator.
+
+## Applications
+Probability itself is really just a tool you can use to help solve problems with some sort of uncertainty. The uncertainty may be due to events that are out of your control or maybe you can have some influence on the outcome. The applications of probability theory are endless and applicable in nearly any field.
```

Comment:
Can you provide any pictures showing how control can be applied?

Reply:
Added a figure on the kalman filter for state estimation